### PR TITLE
sitemap improvements #226

### PR DESCRIPTION
**Pull request by lucadelu**
renamed hugo sitemap and created new one including hugo and manual sitemaps

### CODE REVIEW

File: static/files/sitemap.xml [hunk 1]
```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0" encoding="UTF-8"?>
+   <sitemapindex xmlns="http://www.sitemaps.org/schemas/sitemap/0.9">
+   <sitemap>
+      <loc>https://grass.osgeo.org/sitemap_hugo.xml</loc>
+   </sitemap>
+   <sitemap>
+      <loc>https://grass.osgeo.org/grass78/manuals/sitemap_manuals.xml</loc>
+   </sitemap>
+   </sitemapindex>
```
Comment (neteler) [inline static/files/sitemap.xml:9]:
```suggestion
   <sitemap>
      <loc>https://grass.osgeo.org/grass78/manuals/addons/sitemap_manuals.xml</loc>
   </sitemap>
</sitemapindex>
```

Also consider addon manual pages

**Comment by neteler** [comment]
Requires merge of https://github.com/OSGeo/grass-addons/pull/348

File: static/files/sitemap.xml [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+   <sitemap>
+      <loc>https://grass.osgeo.org/sitemap_hugo.xml</loc>
+   </sitemap>
+   <sitemap>
+      <loc>https://grass.osgeo.org/grass78/manuals/sitemap_manuals.xml</loc>
+   </sitemap>
+   <sitemap>
+      <loc>https://grass.osgeo.org/grass78/manuals/addons/sitemap_manuals.xml</loc>
+   </sitemap>
+</sitemapindex>
```
Comment (neteler) [inline static/files/sitemap.xml:12]:
```suggestion
   <sitemap>
      <loc>https://grass.osgeo.org/grass79/manuals/sitemap_manuals.xml</loc>
   </sitemap>
</sitemapindex>
```
Let's include G79 as well

**Comment by lucadelu** [comment]
Maybe it could be better to use only  https://grass.osgeo.org/grass-stable/manuals/ so we need not to update every time we change version?

**Comment by neteler** [comment]
Fine for me

**Comment by lucadelu** [comment]
Could be merged

File: static/files/sitemap.xml [hunk 3]
```diff
@@ -0,0 +1,12 @@
...
+   <sitemap>
+      <loc>https://grass.osgeo.org/sitemap_hugo.xml</loc>
+   </sitemap>
+   <sitemap>
+      <loc>https://grass.osgeo.org/grass-stable/manuals/sitemap_manuals.xml</loc>
+   </sitemap>
+   <sitemap>
+      <loc>https://grass.osgeo.org/grass-stable/manuals/addons/sitemap_manuals.xml</loc>
+   </sitemap>
+</sitemapindex>
```
Comment (neteler) [inline static/files/sitemap.xml:12]:
```suggestion
   <sitemap>
      <loc>https://grass.osgeo.org/grass-devel/manuals/sitemap_manuals.xml</loc>
   </sitemap>
</sitemapindex>
```

**Pull request merged**